### DISCUSSION TITLE
How to disable sending keycodes to terminal input

### DISCUSSION THREAD

**Discussion by nowskida** [kovidgoyal/kitty#6415] [General]
Hi, 
When I press <Super> + some key in kitty terminal it prints keycode to my shell. 
For example pressing <Super> + E  
![image](https://github.com/kovidgoyal/kitty/assets/100798238/e159f62e-9c74-48e0-9c25-97eb9e10a2bb)

Is there any way I can disable this behavior?

**Comment by kovidgoyal** [comment]
You can map the keypresses you want to eliminate to discard in
kitty.conf

**Reply by nowskida** [reply]
Thanks so much for your reply and for your work :) That what I was thinking I need to do. I was hoping that there is some magic setting that will discard all of those events.
